### PR TITLE
Created Empty Profile Pages

### PR DESCRIPTION
**Pull request by jbhesania**
Set up profile pages that do not display any information other than the name of the user. These pages are found using the url /users/name. This is where the "follow" button will be created.

### CODE REVIEW

**Review by cariygan — APPROVED** [review]
Looks good I think!

File: src/test/java/codeu/controller/ProfilePageServletTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  }
+
+  @Test
+  public void testDoGet() throws IOException, ServletException {
+
+    UserStore mockUserStore = Mockito.mock(UserStore.class);
+    profilePageServlet.setUserStore(mockUserStore);
+    User mockUser = Mockito.mock(User.class);
+
+    Mockito.when(mockRequest.getRequestURI()).thenReturn("/users/testuser");
```
Comment (Linda-Yang) [inline src/test/java/codeu/controller/ProfilePageServletTest.java]: this is probably something really small but it says get URI here and Im not sure if thats a typo or not

**Review by Linda-Yang — COMMENTED** [review]
It looks good! However I am kind of worried about the test case where the username is null/user is not signed in. Or what happens when the username is not valid/does not exist..?

File: src/main/java/codeu/controller/ProfilePageServlet.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package codeu.controller;
+
+import codeu.model.data.User;
```
Comment (saveen-sahni) [inline src/main/java/codeu/controller/ProfilePageServlet.java:3]: Test

**Pull request merged**